react/SearchPage.js

```diff
@@ -1,19 +1,17 @@
-import React, {Component} from 'react'
-import PropTypes from 'prop-types'
+import React, { Component } from "react";
+import PropTypes from "prop-types";
```

Comment:
is your eslint configured correctly ? 
strings should be single-quoted.
no semicolons.